dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -1015,7 +1015,7 @@ public Entry getPublishedEntryByPath(@ApiParam(value = "repository path", requir
     @Timed
     @UnitOfWork(readOnly = true)
     @Path("/path/{repository}")
-    @Operation(operationId = "getAllWorkflowByPath", description = "Get a list of workflows by path.", security = @SecurityRequirement(name = OPENAPI_JWT_SECURITY_DEFINITION_NAME))
+    @Operation(operationId = "getAllWorkflowByPath", description = "Get a list of workflows by path. Do not include workflow name.", security = @SecurityRequirement(name = OPENAPI_JWT_SECURITY_DEFINITION_NAME))
```

Comment:
This says `Do not include`; the next line says `Does not include` -- they should be the same; the way they are now they mean different things.